packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -714,13 +714,17 @@ export class CollaborationWindow {
 	// reference segment for this client
 	currentSeq = 0;
 
+	// Grouped batching means seqNum can potentially be the same for different ops. Use clientSeqNum to differentiate
+	currentClientSeq? = 0;
```

Comment:
does this need to be optional? client seq has always existed, so starting to read it should be fine

Reply:
That's technically a breaking change since `CollaborationWindow` is exported